README.md

```diff
@@ -27,16 +27,17 @@ There are development builds available [here](https://github.com/PrismLauncher/P
 
 Portable builds are provided for Linux, Windows, and macOS.
 
-For Arch, Debian and Gentoo, respectively, you can use these packages to get compiled development versions:
+For Arch, Debian, Fedora or Opensuse and Gentoo, respectively, you can use these packages to get compiled development versions:
```

Comment:
maybe specify openSUSE Tumbleweed since copr only supports Leap up to 15.3 (latest is 15.4) and builds aren't enabled for that

cool pr by the way 😎

Reply:
Will do tomorrow then can't do that on mobile. (actually I can but I hate cli on iPad)